pkg/chunk/purger/purgeplan/delete_plan.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package purgeplan;
```

Comment:
Why extra package just for protobuf file?

Reply:
I wanted to keep it isolated but I think this won't be needed outside of purger package so I think I can move it there.